yax/artifacts/tax_id_data.py

```diff
@@ -0,0 +1,13 @@
+from yax.state.type.artifact import Artifact
+from yax.shared.utilities import taxidtool
+
+
+class TaxIDData(Artifact):
+    def __init__(self, completed):
+        self.tax_id_data = None
+
+        if completed:
+            pass
```

Comment:
yes, both the exact fp for writing and then for reading here have been updated accordingly